### PR TITLE
[1.x] Modify national card validation

### PR DESCRIPTION
**Pull request by Mohammadreza-73**

### CODE REVIEW

File: tests/ValidNationalCardTest.php [hunk 1]
```diff
@@ -24,10 +24,9 @@ protected function setUp(): void
     public function national_code_is_valid()
     {
         $rules = ['national_card' => [new ValidNationalCard()]];
```
Comment (milwad-dev) [inline tests/ValidNationalCardTest.php:26]: Change it to:
```php
$rules = ['national_card' => [new ValidNationalCard()]];
$data = ['national_card' => '0151016437'];
$passes = $this->app['validator']->make($data, $rules)->passes();

$this->assertTrue($passes);
```

File: tests/ValidNationalCardTest.php [hunk 2]
```diff
@@ -40,9 +39,11 @@ public function national_code_is_valid()
     public function national_code_is_not_valid()
     {
         $rules = ['national_card' => [new ValidNationalCard()]];
```
Comment (milwad-dev) [inline tests/ValidNationalCardTest.php:41]: Change it to:
```php
$rules = ['national_card' => [new ValidNationalCard()]];

$validator = $this->app['validator']->make(['national_card' => '0151016438'], $rules)->passes();
$this->assertFalse($validator);

$validator = $this->app['validator']->make(['national_card' => '101010'], $rules)->passes();
$this->assertFalse($validator);
```

**Comment by milwad-dev** [comment]
Why `0151016438` is not true ?!

**Comment by milwad-dev** [comment]
This PR is related to https://github.com/milwad-dev/laravel-validate/issues/15 issue.

**Review by Mohammadreza-73 — COMMENTED** [review]
OK

File: tests/ValidNationalCardTest.php (hunk 1 above)
Comment (Mohammadreza-73) [reply tests/ValidNationalCardTest.php]: changed

**Comment by milwad-dev** [comment]
Thanks!

**Pull request merged**